src/datatables/datatable.ts

```diff
@@ -1,24 +1,33 @@
-import {Component, Input, ChangeDetectorRef, ContentChildren, QueryList, ElementRef, Renderer, HostBinding} from '@angular/core';
+import {Component, Input, ChangeDetectorRef, ContentChildren, QueryList, ElementRef, Renderer, HostBinding, Output, EventEmitter} from '@angular/core';
 import {Subscription} from 'rxjs/Rx';
 import {NglDatatableColumn} from './column';
+import {NglInternalDatatableHeadCell} from './_head';
 import {NglInternalDatatableCell} from './_cell';
 
+export interface INglDatatableSort {
+  key: string;
+  sort: 'asc' | 'desc';
```

Comment:
I think that `dir` or `direction` would be more suitable than `sort` for the sort direction.